modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/test-invalid-bound-object-definitions.json

```diff
@@ -0,0 +1,680 @@
+				"system": true,
+				"type": "Long",
+				"unique": false
+			},
+			{
+				"DBType": "Date",
+				"businessType": "Date",
+				"externalReferenceCode": "b9a086f4-e3bf-1cb5-f1b7-1f70573b161d",
+				"indexed": false,
+				"indexedAsKeyword": false,
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "Modified Date"
+				},
+				"localized": false,
+				"name": "modifiedDate",
+				"objectFieldSettings": [
+				],
+				"readOnly": "true",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "Date",
+				"unique": false
+			},
+			{
+				"DBType": "String",
+				"businessType": "Text",
+				"externalReferenceCode": "219a9309-e879-6dfd-426f-e622670cf091",
+				"indexed": false,
+				"indexedAsKeyword": false,
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "Status"
+				},
+				"localized": false,
+				"name": "status",
+				"objectFieldSettings": [
+				],
+				"readOnly": "true",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "String",
+				"unique": false
+			}
+		],
+		"objectFolderExternalReferenceCode": "default",
+		"objectLayouts": [
+		],
+		"objectRelationships": [
+			{
+				"deletionType": "cascade",
+				"edge": true,
+				"externalReferenceCode": "fbbbac7e-677b-6308-84e1-621e50e84499",
+				"label": {
+					"en_US": "RootToBranch"
+				},
+				"name": "rootToBranch",
+				"objectDefinitionExternalReferenceCode1": "root",
+				"objectDefinitionExternalReferenceCode2": "bd9fd224-2ac4-68b7-85f9-2ce7fb1a6908",
+				"objectDefinitionModifiable2": true,
+				"objectDefinitionName2": "Branch",
+				"objectDefinitionSystem2": false,
+				"objectField": {
+					"DBType": "Long",
+					"businessType": "Relationship",
+					"externalReferenceCode": "17649c5c-7758-06e0-f4f2-b18e050349a2",
+					"indexed": true,
+					"indexedAsKeyword": false,
+					"indexedLanguageId": "",
+					"label": {
+						"en_US": "RootToBranch"
+					},
+					"localized": false,
+					"name": "r_rootToBranch_c_rootId",
+					"objectFieldSettings": [
+						{
+							"name": "objectDefinition1ShortName",
+							"value": "Root"
+						},
+						{
+							"name": "objectRelationshipERCObjectFieldName",
+							"value": "r_rootToBranch_c_rootERC"
+						}
+					],
+					"readOnly": "false",
+					"readOnlyConditionExpression": "",
+					"relationshipType": "oneToMany",
+					"required": true,
+					"state": false,
+					"system": false,
+					"type": "Long",
+					"unique": false
+				},
+				"parameterObjectFieldId": 0,
+				"parameterObjectFieldName": "",
+				"reverse": false,
+				"system": false,
+				"type": "oneToMany"
+			}
+		],
+		"objectValidationRules": [
+		],
+		"objectViews": [
+		],
+		"panelCategoryKey": "",
+		"parameterRequired": false,
+		"pluralLabel": {
+			"en_US": "roots"
+		},
+		"portlet": true,
+		"restContextPath": "/o/c/roots",
+		"rootObjectDefinitionExternalReferenceCode": "root",
```

Comment:
@G-Sa should we follow the same pattern from the other json files? something like "`TESTROOTBJECTDEFINITION`"